app/ui/src/main/java/com/fsck/k9/ui/K9Drawer.java

```diff
@@ -48,9 +64,11 @@
     private int iconFolderResId;
 
     private final List<Long> userFolderDrawerIds = new ArrayList<>();
+    private final List<Long> userAccountDrawerIds = new ArrayList<>();
     private boolean unifiedInboxSelected;
     private String openedFolderServerId;
 
+    private final Preferences preferences = DI.get(Preferences.class);
```

Comment:
Ideally `K9Drawer` would be more like a dumb view. Pass in the accounts, folders, and information which of them are selected. And when items are clicked on, notify observers via callback. That'll make for a clear separation between behavior and presentation.